src/mongocxx/test/collection.cpp

```diff
@@ -35,14 +35,14 @@ TEST_CASE("collection copy", "[collection]") {
 
     std::string collname{"foo"};
     std::string collname2{"bar"};
-    collection coll = db[collname];
+    collection coll = db[stdx::string_view{collname}];
```

Comment:
Is this actually necessary? I don't want it to be - passing a `std::string` to the `database::operator[]` really should work. Otherwise, really hard on the users of the classes.